### PR TITLE
Personal portfolio milestone 10: Modalpopup

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## In this Milestone I was able to:
- Learn and implement Modal popups for my projects sections
- Practice Javascript events
- Dynamically add HTML to my project using Javscript
## Known issue:
- The close button works when you click just below it by a little bit despite redoing the whole styling and Javascript for that part. It however closes when you tap it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,10 +202,11 @@ <h3>Multi-Post Stories</h3>
...
               </ul>
-              <button type="button" class="button">See Project</button>
+              <button type="button" id="muiltipostbtn" class="button">See Project</button>
             </div>
           </div>
         </div>
+        <div class="featuredmodal"></div>
       </section>
 
       <section class="myProjects">
```
Comment (andersonlebon) [inline index.html:212]: - [x] Nice job :clap: on using Javascript to popup a window containing information however The requirements say that all the code concerning `myProjects` should be created using Javascript and be rendered when the page loads. This 👇  is what the requirements state
 
> The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.** Check examples of students' projects:  [here](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [here 👇 ](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.)

File: scripts/modal.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    html: 'html',
+    bootstrap: 'bootstrap',
+    ruby: 'Ruby On Rails',
+    projectImage: './images/mobilemodal.png',
+    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
+    projectbtnlive: 'https://nemwel-boniface.github.io/personal-portfolio/',
+    projectbtnsource: 'https://nemwel-boniface.github.io/personal-portfolio/',
+  },
+];
+
```
Comment (andersonlebon) [inline scripts/modal.js]: - [x] Nice job 👏  on storing your projects in an array of objects however I can only see two projects' information. All of your 6 projects should be using information or data stored in your array of objects(const projectList=[{},{},{}]),

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Nemwel-Boniface 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Nemwel-Boniface) [reply index.html:212]: Hello @andersonlebon thank you for this review. 
I have made the necessary changes

File: scripts/modal.js (hunk 2 above)
Comment (Nemwel-Boniface) [reply scripts/modal.js]: Hello @andersonlebon  thank you for this review.
- I was at first looping through all the buttons to select the details of the modal popup from the array's object.
- I have however made the required changes

**Review by mmsesay — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**